pkgs/development/libraries/opencolorio/default.nix

```
@@ -1,47 +1,28 @@
-{ stdenv, lib, fetchFromGitHub, cmake, boost, pkg-config, lcms2, tinyxml, git }:
+{ stdenv, lib, fetchFromGitHub, cmake, expat, libyamlcpp, ilmbase, pystring, lcms2, python3Packages }:
 
 with lib;
 
 stdenv.mkDerivation rec {
   pname = "opencolorio";
-  version = "1.1.1";
+  version = "2.0.1";
 
   src = fetchFromGitHub {
-    owner = "imageworks";
+    owner = "AcademySoftwareFoundation";
     repo = "OpenColorIO";
     rev = "v${version}";
-    sha256 = "12srvxca51czpfjl0gabpidj9n84mw78ivxy5w75qhq2mmc798sb";
+    sha256 = "194j9jp5c8ws0fryiz936wyinphnpzwpqnzvw9ryx6rbiwrba487";
   };
 
-  outputs = [ "bin" "out" "dev" ];
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ expat libyamlcpp ilmbase pystring lcms2 python3Packages.pybind11 ];
```

> Right now, OCIO2 is compiled without GPU rendering support, but it should be possible to expose it as an optional feature if we want to.

Darwin build is broken for me because it detects GLUT in the regular system paths and attempts a build with GPU rendering support. Is there a CMake flag to explicitly disable GPU support?

If you want to work on enabling GPU support instead, the derivation needs `Carbon`, `GLUT` and `Cocoa` on Darwin to build.

```suggestion
  buildInputs = [
    expat libyamlcpp ilmbase pystring lcms2 python3Packages.pybind11
  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
    Carbon GLUT Cocoa
  ];
```

(In `all-packages.nix`, do `inherit (darwin.apple_sdk.frameworks) Carbon GLUT Cocoa;` for opencolorio2)

---

Can confirm this fixes the build
```diff
diff --git a/pkgs/development/libraries/opencolorio/default.nix b/pkgs/development/libraries/opencolorio/default.nix
index d9de24b7fa0..573d5f62cd6 100644
--- a/pkgs/development/libraries/opencolorio/default.nix
+++ b/pkgs/development/libraries/opencolorio/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, lib, fetchFromGitHub, cmake, expat, libyamlcpp, ilmbase, pystring, lcms2, python3Packages }:
+{ stdenv, lib, fetchFromGitHub, cmake, expat, libyamlcpp, ilmbase, pystring, lcms2, python3Packages, Carbon, GLUT, Cocoa }:
 
 with lib;
 
@@ -14,7 +14,11 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ cmake ];
-  buildInputs = [ expat libyamlcpp ilmbase pystring lcms2 python3Packages.pybind11 ];
+  buildInputs = [
+    expat libyamlcpp ilmbase pystring lcms2 python3Packages.pybind11
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    Carbon GLUT Cocoa
+  ];
 
   cmakeFlags = [ "-DOCIO_INSTALL_EXT_PACKAGES=NONE" ];
 
diff --git a/pkgs/top-level/all-packages.nix b/pkgs/top-level/all-packages.nix
index 42306cf5b27..2877c8da84d 100644
--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -17514,7 +17514,9 @@ in
   openldap = callPackage ../development/libraries/openldap { };
 
   opencolorio1 = callPackage ../development/libraries/opencolorio/1.x.nix { };
-  opencolorio2 = callPackage ../development/libraries/opencolorio { };
+  opencolorio2 = callPackage ../development/libraries/opencolorio {
+    inherit (darwin.apple_sdk.frameworks) Carbon GLUT Cocoa;
+  };
   opencolorio = opencolorio1;
 
   opendmarc = callPackage ../development/libraries/opendmarc { };
```

---

As there is no way to disable GPU rendering, I added the necessary dependencies for it on Linux and Darwin.